### DISCUSSION TITLE
Needing help from programmers

### DISCUSSION THREAD

**Discussion by ghost** [community/community#138056] [Programming Help]
### Body

Hey everyone! I'm considering launching a business and I would love to have your input. I'm planning to establish a free searchable database that allows users to search for the ideal dog food, treats, and or toys for their beloved doggies based on their requirements. If anyone is interested in joining this project, please reply to this post. I'm not sure if this idea is too ambitious, so your feedback would be greatly appreciated! (I want to build it in Flask)

Thinking of naming it "justadoggie.com"

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by phuyalgaurav** [comment]
Hey, i maybe intrested. What are the technologies you are gonna use?

**Comment by ArtyomCZ** [comment]
Rule number one: never reveal which domain you want to buy until you've secured it.
